Regression/Checksum/benchmarks_json/space_charge_initialization.json

```diff
@@ -0,0 +1,30 @@
+{
+  "beam": {
+    "particle_Bx": NaN,
```

Comment:
Did you notice these NaNs?

Reply:
I did not, thanks for pointing this out, I am afraid this is due to `cfl = 1.e-15` (giving a `/0` somewhere). I am not sure how to handle this. Maybe just do a normal time step and hope things don't change too much.